pymatgen/io/qchem/inputs.py

```diff
@@ -536,6 +586,26 @@ def geom_opt_template(geom_opt: dict) -> str:
         geom_opt_list.append("$end")
         return "\n".join(geom_opt_list)
 
+    @staticmethod
+    def pcm_nonels_template(pcm_nonels: dict) -> str:
+        """
+        Pcm run template.
+
+        Args:
+            pcm ():
+
+        Returns:
+            (str)
+        """
+        pcm_nonels_list = []
+        pcm_nonels_list.append("$pcm_nonels")
+        for key, value in pcm_nonels.items():
+            # if the value is None, don't write it to output
```

Comment:
If you're specifically trying to avoid None, again, `is not None` is the better way to do it.